samples/nextjs/src/lib/page-props-factory.ts

```diff
@@ -122,18 +113,22 @@ export class SitecorePagePropsFactory {
           isServerSidePropsContext(context) ? (context as GetServerSidePropsContext).req : undefined,
           isServerSidePropsContext(context) ? (context as GetServerSidePropsContext).res : undefined
         )
-        .catch((error: AxiosError<LayoutServiceData>) => {
-          if (error.response?.status === 404) {
-            // Let 404s (invalid path) through.
-            // layoutData.sitecore.route will be missing, but
-            // layoutData.sitecore.context will provide valuable information
-            notFound = true;
-            return error.response.data;
+        .catch((error) => {
+          if (error.isAxiosError) {
```

Comment:
@ambrauer Typescript doesn't say anything that `isAxiosError` doesn't exist, so error has type of `AxiosError`. I believe in the future we can't use this check, because GraphqlLayoutService will not throw AxiosError, so need to have type check here.

Reply:
@illiakovalenko This is removed as you realized in separate PR #559 (which has dependency on this one), so this will be resolved there.